openhtf/io/proto/generate_units.py

```diff
@@ -0,0 +1,70 @@
+"""Generates util/units.py from units.proto!
```

Comment:
oh god, this is moderately painful... but probably the best way to do this, in fact i'm impressed at how not-hacky you managed to make this.

one thing i'd like to change, though, if you don't mind a little extra work ;)  if we're going to generate units.py, we should probably do so in the build process and not have a units.py checked in to the repo.  you could roll it into build_proto if you want to save some boilerplate (i don't think we need a build_units target specifically for just this, but entirely up to you).